jython/DispatcherSystem/CreateIcons.py

```diff
@@ -485,6 +500,23 @@ def removeSensors(self):
             #print 'remove sensor {}'.format(item.getDisplayName())
             sensors.deleteBean(item, 'DoDelete')
 
+        #remove unused sensors (might not be needed now but was at one stage)
+        deleteList = []
+        for sensor in sensors.getNamedBeanSet():
+            sysname = sensor.getSystemName()
+
+            if sysname.startswith("IS:DSMP:"):
```

Comment:
You should not need to do the DSMT and DSMP sensors since they are already included in the delete sensors section.

"removeSensors" is also where the direction sensors should be deleted.

Reply:
I had DSMT and DSMP sensors where the username was null. I realise I should not have needed to do the DSMT and DSMP sensors, but I did. Whether this is now needed is debateable. If you prefer we can delete for now, and see whether the problem re-appears.

Reply:
I don't see a problem leaving the DSMT and DSMP without user name code.  It should not hurt anything.